pallets/randomness-beacon/src/lib.rs

```diff
@@ -330,7 +332,7 @@ pub mod pallet {
 			DidUpdate::<T>::put(true);
 
 			// dispatch pulses to subscribers
-			T::Dispatcher::dispatch(pulses)?;
+			// T::Dispatcher::dispatch(pulses)?;
```

Comment:
We could make it compatible for the time being (until the dispatcher is changed)
```suggestion
			// T::Dispatcher::dispatch([pulse])?;
```